windows/src/developer/TIKE/child/UfrmDebug.pas

```diff
@@ -743,31 +822,31 @@ procedure TfrmDebug.ExecuteEventAction(n: Integer);
         // Extended Key State
         flag := flag or KEYFLAG_EXTENDED;
 
+      state := SaveMemoSelectionState;
```

Comment:
Relates to
(1) Correct handling of text selection and deletion
(3) Deadkey realignment after text changes
(4) Correct tracking of cursor position between events